Client/Frontend/Browser/Search/InitialSearchEngines.swift

```diff
@@ -10,7 +10,16 @@ import Foundation
 class InitialSearchEngines {
     /// Type of search engine available to the user.
     enum SearchEngineID: String {
-        case google, bing, duckduckgo, yandex, qwant, startpage, yahoo
+        case google, bing, duckduckgo, yandex, qwant, startpage, yahoo, ecosia
+        
+        var excludedFromOnboarding: Bool {
```

Comment:
This property is temporary right until we remove the yahoo engine totally ?

Reply:
I think we can keep it, if we add a lot of predefined search engines in the future we can choose which of them to show during onboarding, to avoid having a long list of search engines